themes/theme-gmd/components/ProductGrid/components/Item/index.jsx

```diff
@@ -74,11 +74,11 @@ const Item = ({ product, display, isFavorite }) => (
         {/* PRICE - STRIKE PRICE - PRICE INFO */}
         {(!display || display.price) && (
           <Fragment>
-            <Portal name={portals.PRODUCT_ITEM_PRICE_BEFORE} props={{ productId: product.id }} />
-            <Portal name={portals.PRODUCT_ITEM_PRICE} props={{ productId: product.id }}>
+            <Portal name={portals.PRODUCT_ITEM_PRICE_BEFORE} props={{ productId: product.id,location: 'productGrid' }} />
```

Comment:
there should be a space after prooduct.id,

Reply:
fixed